qutebrowser/browser/commands.py

```diff
@@ -965,6 +978,32 @@ def tab_move(self, index: typing.Union[str, int] = None,
         cur_idx = self._current_index()
         cmdutils.check_overflow(cur_idx, 'int')
         cmdutils.check_overflow(new_idx, 'int')
+
+        if config.val.tabs.tree_tabs:
+            new_idx += 1
```

Comment:
Could you add a comment saying this is due to the root taking up slot 0? 

Better yet, can you add a flag to `traverse` to skip the root, so we don't need to worry about it?